index.html

```diff
@@ -231,6 +231,30 @@ <h3>Languages</h3>
         <img src="assets/vector.svg " alt="">
       </div>
   </section>
+
+  <section class="contact-me">
+    
+    <form action="https://formspree.io/f/{form_id}" method="POST">
+      <div class="background">
+      <h2>contact-me</h2>
```

Comment:
As the previous reviewer mentioned, kindly fix the action of your form. You are using the example placeholder (https://formspree.io/f/{form_id}), so your form is not actually getting sent. In order for other users to communicate with you, you will need to create an actually formspree form_id. You can do so by creating an account at https://formspree.io/ and then creating a new form on that page. After that, you can use your form endpoint in your action.

![formspree](https://github.com/santitoot/finish-mobile-version/assets/111214189/a8d04920-942d-49a0-9deb-22eb77dbc254)

This way, other people that visit your portfolio can communicate with you and your portfolio will be fully functional 👍